### DISCUSSION TITLE
System audio not working

### DISCUSSION THREAD

**Discussion by Josh65-2201** [MichaIng/DietPi#6746] [Q&A] [answered]
I have spent a while trying to get audio working. The main error given is that `the pulseaudio server isn't running` if I install it then it shows as working. However programs like vlc won't output audio, using the command line version it spams `pulseaudio error, overflow, flushing`

**Comment by MichaIng** [comment]
Why are you using PulseAudio at all? We setup plain ALSA when selecting a sound card via `dietpi-config` audio options. If this (without PulseAudio) does not work, please tell us some more info about your system, at best open a bug report (below Issues) and fill the template. And add the output of these two commands:
```sh
aplay -l
speaker-test
```

**Reply by Josh65-2201** [reply, marked as answer]
Ok so i was using the xefc desktop environment witch has a volume mixer that needs pluseaudio installed,  as VLC wasn't outputting audio i thought I needed to install it. I have since uninstalled pulseaudio and ran the commands you shows which outputed audio and vlc is now working.

**Reply by MichaIng** [reply]
Okay great. Probably there is a ALSA mixer panel plugin as well. Otherwise form console: `alsamixer`. Or as GUI application:
```sh
apt install alsamixergui
```
It should appear in the application menu then.
